sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -259,7 +264,7 @@ abstract class BaseHashJoinIterator(
     // TODO: Replace this estimate with exact join row counts using the corresponding cudf APIs
     //       being added in https://github.com/rapidsai/cudf/issues/9053.
     joinType match {
-      case _: InnerLike | LeftOuter | RightOuter =>
+      case _: InnerLike | LeftOuter | RightOuter | ExistenceJoin(_) =>
```

Comment:
ExistenceJoin cannot explode like Inner/LeftOuter/RightOuter.  This should be treated more like a semi join, as the number of output rows should be the same as the number of left table input rows.